utils/config/config.go

```
@@ -233,29 +233,52 @@ func saveConfig(config *ConfigV3) error {
 	return nil
 }
 
-func readConf() (*ConfigV3, error) {
+func readConf() (*ConfigV4, error) {
 	confFilePath, err := getConfFilePath()
 	if err != nil {
 		return nil, err
 	}
-	config := new(ConfigV3)
 	exists, err := fileutils.IsFileExists(confFilePath, false)
 	if err != nil {
 		return nil, err
 	}
-	if !exists {
-		return config, nil
-	}
-	content, err := fileutils.ReadFile(confFilePath)
-	if err != nil {
-		return nil, err
-	}
-	if len(content) == 0 {
-		return new(ConfigV3), nil
-	}
-	content, err = convertIfNeeded(content)
-	if err != nil {
-		return nil, err
+	config := new(ConfigV4)
+	var content []byte
+	if exists {
+		content, err = fileutils.ReadFile(confFilePath)
+		if err != nil {
+			return nil, err
+		}
+		if len(content) == 0 {
+			return config, nil
+		}
+	} else {
+		// Try to look up for older config files
+		currentVersion, _ := strconv.Atoi(coreutils.GetConfigVersion())
```

Let's not ignore the error returned from ```strconv.Atoi```.